src/libraries/System.Net.Http/tests/TrimmingTests/HttpClientTest.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+class Program
+{
+    static async Task<int> Main(string[] args)
```

Comment:
I don't understand the intention of this test. I also don't understand why a trimming test is being included in a change concerning the Platform Compatibility Analyzer.

Reply:
We want to trim System.Net.Quic.dll as well.